.github/workflows/brokenLinks.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+name: Check markdown links
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1
```

Comment:
I think you need to specify where the files are for this to work correctly?

https://github.com/lycheeverse/lychee#commandline-usage